### ISSUE TITLE
How do I package this with pyinstaller?

### ISSUE THREAD

**Issue by taytanqbpass** [kaliiiiiiiiii/Selenium-Driverless#95]
Hey there,

I tried to package this with my executable by using "pyinstaller -F --icon icon.ico --hidden-import=selenium_driverless main.py" but I'm getting the error: 
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\admin\\AppData\\Local\\Temp\\_MEI145002\\selenium_driverless/files/is_first_run'

any help would be appreciated, Thanks.

**Comment by kaliiiiiiiiii** [comment]
Yep, you'll have to include the full driverless packagr directory.

**Comment by milahu** [comment]
see also #122

**Comment by phuctran1203** [comment]
@Acanis87 thanks bro, you saved me. AI dump af.
